### PR TITLE
remove storage platform restrictions and fix image cutout docs

### PR DESCRIPTION
**Pull request by LinhuAdobe**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -70,13 +66,7 @@ If you are not currently a member, please sign up at [https://photoshop.adobelan
 
 The Adobe Photoshop APIs will allow you to make both layer and document level edits to Photoshop PSD files.  This page is meant to help you onboard with the service and get you started with some basic usage examples.
 
-The API documentation is published at
-
-[Photoshop API Documentation](https://adobedocs.github.io/photoshop-api-docs/)
-
-[Lightroom API Documentation](https://adobedocs.github.io/lightroom-api-docs/)
-
-[Image Cutout API Documentation](https://adobedocs.github.io/photoshop-api-docs/#api-Sensei-ImageCutout)
```
Comment (shayanc1985) [inline README.md]: Not sure if the other links need to be removed. Can you check with Morgan/Archy?

File: README.md [hunk 2]
```diff
@@ -1014,15 +916,3 @@ The typical workflow involves making a synchronous API call to the POST endpoint
...
-
-The Adobe Lightroom APIs allow you to make Lightroom-like automated edits to image files.
-
-## General Workflow
-
-The typical workflow involves making an API POST call to the endpoint https://image.adobe.io/lrService/ for which the response will contain a link to check the status of the asynchronous job. Making a GET call to this link will return the status of the job and, eventually, the links to your generated output.
-
-## How to use the API's
-
-The API's are documented at [https://github.com/AdobeDocs/lightroom-api-docs](https://github.com/AdobeDocs/lightroom-api-docs/)
```
Comment (shayanc1985) [inline README.md]: Same here. These should not be removed.

File: README.md (hunk 1 above)
Comment (LinhuAdobe) [reply README.md]: fixed

File: README.md (hunk 2 above)
Comment (LinhuAdobe) [reply README.md]: fixed

**Pull request closed without merging**